docs/provider/daemon/getting_started/run_remote.md

```diff
@@ -0,0 +1,131 @@
+    $ export TDEX_SSL_KEY=~/.tdex-daemon/trade-tls/privatekey.pem
+    $ export TDEX_SSL_CERT=~/.tdex-daemon/trade-tls/fullchain.pem
+
+    # Start a daemon with TLS enabled ONLY on Trade interface
+    $ tdexd
+    ```
+- Docker:
+    ```sh
+    # Start a dockerized daemon with TLS enabled ONLY on Trade interface
+    $ docker run -it -d \
+    --name tdexd \
+    --restart unless-stopped \
+    -p 9945:9945 -p 9000:9000 \
+    -v `pwd`/tdexd:/.tdex-daemon \
+    # Here, it's required to mount the filepaths of the TLS key/cert as volumes
+    # of the container
+    -v `pwd`/privatekey.pem:/privatekey.pem \
+    -v `pwd`/fullchain.pem:/fullchain.pem \
+    -e TDEX_SSL_KEY=/privatekey.pem \
+    -e TDEX_SSL_CERT=/fullchain.pem \
+    ghcr.io/tdex-network/tdexd:latest
+    ```
+
+:::tip
+Enable secure connection on both interfaces
+
+- Standalone:
+    ```sh
+    $ export TDEX_OPERATOR_EXTRA_IP=54.170.20.39
+    $ export TDEX_OPERATOR_EXTRA_DOMAIN=provider.mydomain.com
+    $ export TDEX_SSL_KEY=~/.tdex-daemon/trade-tls/privatekey.pem
+    $ export TDEX_SSL_CERT=~/.tdex-daemon/trade-tls/fullchain.pem
+
+    # Start a daemon with macaroons/TLS over Operator interface AND TLS enabled 
+    # over Trade interface
+    $ tdexd
+    ```
+- Docker:
+    ```sh
+    # Start a dockerized daemon with macaroons/TLS enabled over Operator
+    # interface AND TLS enabled over Trade interface
+    $ docker run -it -d \
+    --name tdexd \
+    --restart unless-stopped \
+    -p 9945:9945 -p 9000:9000 \
+    -v `pwd`/tdexd:/.tdex-daemon \
+    -v `pwd`/privatekey.pem:/privatekey.pem \
+    -v `pwd`/fullchain.pem:/fullchain.pem \
+    -e TDEX_OPERATOR_EXTRA_IP=54.170.20.39 \
+    -e TDEX_OPERATOR_EXTRA_DOMAIN=provider.mydomain.com \
+    -e TDEX_SSL_KEY=/privatekey.pem \
+    -e TDEX_SSL_CERT=/fullchain.pem \
+    ghcr.io/tdex-network/tdexd:latest
+    ```
+:::
+
+## Configure a local Operator CLI to connect with remote daemon
+
+Now that your daemon is up and running on your remote machine, if you made your Operator interface reachable, you can safely interact with it with the CLI on your laptop or home pc.
+
+To do this, you need to import in your local machine a copy of both the macaroon named `admin.macaroon` and the TLS certificate named `cert.pem` that you can find in the daemon's datadir. If, for example you have SSH access to the remote host you can take advantage of the `scp` command:
+
+```sh
+$ scp ubuntu@54.170.20.39:~/.tdex-daemon/macaroons/admin.macaroon ~/.tdex-daemon-remote/admin.macaroon
+$ scp ubuntu@54.170.20.39:~/.tdex-daemon/tls/cert.pem ~/.tdex-daemon-remote/cert.pem
+```
+
+Now it's time to configure the local Operator CLI to connect to the remote daemon:
+
+```sh
+$ tdex config init --macaroons_path ./admin.macaroon --tls_cert_path ./cert.pem --rpc_address provider.mydomain.com:9000
+
+# Now it's possible to start initializing the daemon if it's a brand new one..
+$ tdex genseed
```

Comment:
link to the other docs here?

Reply:
do you mean links to the other pages or an external link to tdex-box which fits for production usage?

Reply:
for instructions on how to use the tdex cli I mean, we can link directly the docs # Configure CLI etc..